src/utils.ts

```diff
@@ -295,7 +295,7 @@ export class Utils {
 
   /** @internal returns the passed element if scrollable, else the closest parent that will, up to the entire document scrolling element */
   static getScrollElement(el?: HTMLElement): HTMLElement {
-    if (!el) return document.scrollingElement as HTMLElement;
+    if (!el) return document.documentElement as HTMLElement;

```

Comment:
need to understand that change as I don't want break other browser for IE, which has been dead for a while really.. so not interested in supporting it unless it's 'free'... 

maybe do || if scrollingElement doesn't exist on IE ?